src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_StackPanel.cs

```diff
@@ -110,12 +110,24 @@ private async Task MaxSizingTest(Size maxConstraints)
 
 				if (!double.IsInfinity(maxConstraints.Width))
 				{
+#if WINDOWS_UWP
+					Assert.AreEqual(constrained.ActualWidth, orientation == Orientation.Horizontal ? 1000 : maxConstraints.Width);
+#else
+					// TODO: Align Uno with Windows behavior.
 					Assert.AreEqual(constrained.ActualWidth, maxConstraints.Width);
+#endif
```

Comment:
Note: This PR doesn't change the asserts for Uno as the behavior for this test hasn't changed. But I'm adding a `#if` for Windows to make the test pass on UWP SamplesApp and make it clear that our behavior doesn't match Windows, in case someone came across this test in future. Just making the difference between Uno and Windows apparent.